root/static/scripts/edit/URLCleanup.js

```diff
@@ -1026,6 +1026,19 @@ const CLEANUPS = {
       return url;
     },
   },
+  'dynamicrangedb': {
+    match: [new RegExp('^(https?://)?dr\\.loudness-war\\.info', 'i')],
+    type: LINK_TYPES.otherdatabases,
+    clean: function (url) {
+      // Seems to support http only
+      url = url.replace(/^(?:https?:\/\/)?dr\.loudness-war\.info\//, 'http://dr.loudness-war.info/');
```

Comment:
Since we're just removing the 's' from https here, do we need the whole `dr\.loudness-war\.info\/` part of the regexp? I'd think we can assume that's already there, if it matches the `match` regexp above.

Reply:
I guess not :)